ios-base/View/Home/HomeRoutes.swift

```diff
@@ -15,9 +15,7 @@ enum HomeRoutes: Route {
   var screen: UIViewController {
     switch self {
     case .home:
-      guard let home = UIStoryboard
-        .instantiateViewController(HomeViewController.self)
-      else {
+      guard let home = R.storyboard.main.homeViewController() else {
```

Comment:
Is this magic?

Reply:
R generates a `struct R` that contains all the magic 👀, but it's recommended to ignore the file on verison control.